build.sbt

```diff
@@ -26,7 +28,15 @@ version in Docker := "1.0.0-SNAPSHOT"
 
 organization := "com.codacy"
 
-val swiftlintVersion = "0.23.1"
```

Comment:
This was unused then?

Reply:
Yup, Marlon must have forgotten the value.